examples/nomad-consul-separate-cluster/main.tf

```diff
@@ -13,6 +13,7 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 provider "aws" {
+  profile = "${var.aws_profile}"
```

Comment:
If you have no profiles defined locally (you always use env vars for AWS auth), will this work correctly?

Reply:
Yes. If the variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are given, but no profile then they are taken to authenticate.